slides/_comments_docstrings.qmd

```diff
@@ -272,3 +272,40 @@ Docstrings:
 
 - Work through the file adding docstrings where they are missing.
 - If you are unsure about variable types or meanings at any point you can sneak a look ahead to the code in exercise 5.
+
+
+## READMEs {.smaller}
+
+- First point of contact for a new user/contributor with the code repository
+- Should give essential information on 
+   - what this is 
+   - what it's for 
+   - how to get started
+- In the best case it also tells you
+   - who built/is building this
+   - how to contribute
+   - how to reuse
+
+## Recommended README Content {.smaller}
+
+ - Overview of software, including relevant papers
+ - Project/release status
+ - Requirements and environment
+ - Installation
+ - Building
+ - Usage/Quick start guide (incl. simple example with expected output)
+ - Documentation and Support: User Guide/Website/Help/Discussion forum
+ - Contributing guide
+ - Authors and Acknowledgement
+ - Licence
+ - Known Issues
```

Comment:
We should probably ensure that our repo readme for this workshop meets most of these in #11 😅

Reply:
I think the most important parts are now there, but it could always be extended....